scipy/ndimage/tests/test_measurements.py

```diff
@@ -15,9 +15,6 @@
          np.float32, np.float64]
 
 
-np.mod(1., 1)  # Silence fmod bug on win-amd64. See #1408 and #1238.
```

Comment:
For clarity, the comment here points to old Trac issues, the GitHub issue numbers are different.

Reply:
See https://github.com/scipy/scipy/issues/1933 instead